The rosdistro validation is reporting related drama in humble:
```
E       AssertionError: ["In file '''/home/runner/work/rosdistro/rosdistro/rolling/distribution.yaml''': Could not validate doc entry for repo ros_gz with error [[[Could not validate repository with url https://github.com/ignitionrobotics/ros_gz.git and version humble from entry at line 4701. Error reason: subprocess call ['git', 'ls-remote', 'https://github.com/ignitionrobotics/ros_gz.git', 'refs/tags/*'] failed: Command '['git', 'ls-remote', 'https://github.com/ignitionrobotics/ros_gz.git', 'refs/tags/*']' returned non-zero exit status 128.]]]"] is not false
```

It would be optimal to get that resolved and merge master into this branch so that CI is clean for merge. Otherwise this release looks correct for rolling.

--- a/rolling/distribution.yaml
+++ b/rolling/distribution.yaml
@@ -4706,10 +4706,16 @@ repositories:
       packages:
       - ros_gz
       - ros_gz_bridge
-      - ros_gz_gazebo
-      - ros_gz_gazebo_demos
       - ros_gz_image
       - ros_gz_interfaces
+      - ros_gz_sim
+      - ros_gz_sim_demos
+      - ros_ign
+      - ros_ign_bridge
+      - ros_ign_gazebo
+      - ros_ign_gazebo_demos
+      - ros_ign_image
+      - ros_ign_interfaces
       tags:
         release: release/rolling/{package}/{version}
       url: https://github.com/ros2-gbp/ros_ign-release.git

--- a/rolling/distribution.yaml
+++ b/rolling/distribution.yaml
@@ -4700,7 +4700,7 @@ repositories:
   ros_gz:
     doc:
       type: git
-      url: https://github.com/ignitionrobotics/ros_gz.git
+      url: https://github.com/gazebosim/ros_gz.git
       version: humble
     release:
       packages:
@@ -4723,8 +4723,8 @@ repositories:
     source:
       test_pull_requests: true
       type: git
-      url: https://github.com/ignitionrobotics/ros_ign.git
-      version: ros2
+      url: https://github.com/gazebosim/ros_gz.git
+      version: humble
     status: developed
   ros_image_to_qimage:
     doc:

